.github/workflows/ci.yaml

```diff
@@ -70,6 +60,14 @@ jobs:
           ..
         make -j2 -k onnx-proto tf-proto tidy
 
+    - name: Save cache files for tidy
+      uses: actions/cache/save@v3 
+      if: always()
```

Comment:
So this will run even if the previous steps have failed, which will match the behavior of the previous always-upload-cache we were using:

https://stackoverflow.com/a/58859404/375343